app/services/idv/steps/ssn_step.rb

```diff
@@ -7,6 +7,10 @@ def call
         return invalid_state_response if invalid_state?
 
         flow_session[:pii_from_doc][:ssn] = flow_params[:ssn]
+
+        if IdentityConfig.store.proofing_device_profiling_collecting_enabled
+          flow_session[:threatmetrix_session_id] = threatmetrix_session_id
```

Comment:
Will we want this to be present in the in-person proofing flow version of the SSN step as well?

Reply:
That is a very good question, I am not 100% certain, we need to confirm

Reply:
**update:** Let's include this on all instances of the SSN page and we can add more specific feature flags if needed